tests/src/test/scala/apigw/healthtests/ApiGwEndToEndTests.scala

```diff
@@ -142,6 +142,72 @@ class ApiGwEndToEndTests
         }
     }
 
+    it should "list api-experimental alphabetically by Base/Rel/Verb and Action name" in withAssetCleaner(wskprops) {
+    (wp, assetHelper) =>
+
+        val actionName1 = "actionName1"
```

Comment:
Use action names that are unique to this test module.. that way it won't accidentally collide with another test.